trappy/filesystem.py

```diff
@@ -65,3 +65,65 @@ class FilesystemExt4SyncFileExit(FilesystemExt4Base):
     """The unique word that will be matched in a trace line"""
 
 register_ftrace_parser(FilesystemExt4SyncFileExit)
+
+def parse_f2fs_data(data_str, equals_parser):
+    data_str = data_str.replace(" = ", "=")
+    data_str = data_str.replace(",", " ")
+    return equals_parser(data_str)
```

Comment:
Can you make a base class and put the parse implementation into the `generate_data_dict` method as you did for the ext4 parser?

Reply:
ah, of course, I should have remembered... fixing up